### PR TITLE
{bio}[dummy/dummy] Kent_tools v20130806

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/k/Kent_tools/Kent_tools-20130806-linux.x86_64.eb [hunk 1]
```diff
@@ -20,6 +20,8 @@ postinstallcmds = [
     "cp -a %(builddir)s/Kent_tools/blat/FOOTER.txt %(installdir)s/bin/FOOTER_blat.txt",
 ]
 
+osdependencies = [('libpng12', 'libpng12-0')]
```
Comment (boegel) [inline easybuild/easyconfigs/k/Kent_tools/Kent_tools-20130806-linux.x86_64.eb]: well, we can do better, and include a dependency on a dummy build of libpng?

Comment (verdurin) [inline easybuild/easyconfigs/k/Kent_tools/Kent_tools-20130806-linux.x86_64.eb]: @boegel - agreed, that would be cleaner and more idiomatic.

**Comment by verdurin** [comment]
As you suggested, I've added a `dummy` build of the latest `libpng` release in the `1.2.x` branch, so as to pick up the latest security fixes, and used that to meet the dependency for `Kent_tools`, instead of using OS packages.

**Comment by boegel** [comment]
@verdurin sadly, this introduces a conflict, i.e. this `libpng` version clashes with the `libpng` used as a dep for Python which is a dep for `i-cisTarget` (just like `Kent_tools`)...

```
Conflict found for dependencies of i-cisTarget-20160602-intel-2016a-Python-2.7.11: libpng-1.2.56 vs libpng-1.6.21-intel-2016a
Conflict found for dependencies of i-cisTarget-20160602-intel-2016a-Python-2.7.11: zlib-1.2.8 vs zlib-1.2.8-intel-2016a
```

Replacing the `libpng` used as a dep for `Python` is not a good option imho.

We *could* make this `libpng` 1.2.x easyconfig a special case, by using a more specific software name like `libpng12`, but then we need to make sure it doesn't shadow any files provided by whichever `libpng` is loaded (or the other way around). Is that feasible?

**Comment by verdurin** [comment]
@boegel - sounds feasible, yes, though I might need your help in avoiding those conflicts.

**Comment by boegel** [comment]
@verdurin it probably boils down to trimming the installation via `postinstallcmds` to remove stuff...

**Comment by boegel** [comment]
@verdurin ping on this?

**Comment by verdurin** [comment]
I've worked around this one locally by using the OS version of the old `libpng` dependency, while I work on a build-from-source version.

If you think this is worth persevering with, I'm happy to do so.

**Comment by migueldiascosta** [comment]
@verdurin, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**